serverless.yml

```diff
@@ -47,16 +47,29 @@ custom:
   dependency-invoke:
     storage: dependencies
     dependencies:
-    - name: cvs-svc-test-results
-      lang: javascript
-      git: git@github.com:dvsa/cvs-svc-test-results.git
-      branch: feature/CVSB-1241
-      handler: src/functions/getTestResultsByTesterStaffId.getTestResultsByTesterStaffId
-      commands:
-      - npm install
-      - lsof -i tcp:8004 | awk 'NR!=1 {print $2}' | xargs kill || true
-      - node_modules/serverless/bin/serverless dynamodb install
-      - "sed -i '' 's/migrate: false/migrate: true/g' serverless.yml"
-      - "sed -i '' 's/seed: false/seed: true/g' serverless.yml"
-      - "sed -i '' 's/noStart: true/noStart: false/g' serverless.yml"
-      - node_modules/serverless/bin/serverless dynamodb start
+      - name: cvs-svc-test-results
+        lang: javascript
+        git: git@github.com:dvsa/cvs-svc-test-results.git
+        branch: feature/CVSB-1241
```

Comment:
why is it checking out the branch `feature/CVSB-1241`, shouldn't the default be the `master` one so that is always up-to-date?

Reply:
I did wonder about that and meant to flag - thought it might be temporary since it won't pass testing with the develop branch at the moment. Not sure how we handle these "need it this way for testing, but different way for prod" scenarios

Reply:
it can be a branch environment variable so that it is corresponding to the environment where it is running.

Reply:
That change is a very old one.
This was temporary, until we have everything merge in develop.
To implement an environment variable impacts also the devops team. Also I am not sure if we can use env variable in serverless.yml file. I will check